apicast/libexec/boot.lua

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env resty
```

Comment:
this should be removed and same changes applied to the keycloak configuration loader